config/prow/plugins.yaml

```diff
@@ -155,6 +155,11 @@ blockades:
   blockregexps:
   - ^examples/
   explanation: "examples/ has moved to https://github.com/kubernetes/examples/"
+- repos:
+  - kubernetes/kubernetes
+  blockregexps:
+  - ^pkg/util/[^/]+\.go$
+  explanation: "Utility code must be added to a subpackage like pkg/util/flag, not pkg/util directly. See: #49993"
```

Comment:
https://github.com/kubernetes/kubernetes/issues/49993

Reply:
er we probably wanted https://github.com/kubernetes/kubernetes/issues/49923, updating.